tests/integration/tier1/checks-after-conversion/test_disable_data_collection.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+import pytest
+
+
+@pytest.mark.disable_data_collection
+def test_disable_data_collection():
```

Comment:
I would also add a test that verifies the opposite, that is that `convert2rhel.facts` is created. It would be run as a default test as part of the `checks-after-conversion` for all except the `activation-key`.

Reply:
Good point!

Reply:
Looking at it now, I have realized we already have this check covered by Rodolfo's `flag_system_as_converted` here https://github.com/oamg/convert2rhel/pull/723/files#diff-0ab2313aa918d5ff6ce259e3237bbeda0871cf808112bf060139298df93f8f37R23
@kokesak

Reply:
Oh, cool then :D